### PR TITLE
feat: release new web ui with new features and look

### LINKED ISSUE #79: [Feature]: UI Button to suspend namespace
Right now you can manually suspend namespace on demand based on changing the annotation.
It would be nice to have an option to do this from UI as it  is easy to see which namespaces are suspended and which are not.

### PR DESCRIPTION
**Pull request by eze-kiel**
this PR closes #79

### CODE REVIEW

File: webui/assets/unsuspend.html [hunk 1]
```diff
@@ -1,35 +1,38 @@
 <!doctype html>
 
-{{ template "head.html" }}
+{{ template "_head.html" }}
```
Comment (xakraz) [inline webui/assets/unsuspend.html:3]: ❓ What are the differences between `webui/assets/unsuspend.html` and `webui/assets/suspend.html`?

File: webui/assets/home.html [hunk 2]
```diff
@@ -1,130 +1,112 @@
...
+            {{if eq .State "Running"}}
+              <a href="/suspend?name={{.Name}}" class="btn btn-danger btn-sm" role="button">Suspend</a>
+            / <a href="/unsuspend?name={{.Name}}" class="btn btn-success btn-sm disabled" role="button" aria-disabled="true">Unsuspend</a>
+            {{else if eq .State "Suspended"}}
+            <a href="/suspend?name={{.Name}}" class="btn btn-danger btn-sm disabled" role="button" aria-disabled="false">Suspend</a>
+            / <a href="/unsuspend?name={{.Name}}" class="btn btn-success btn-sm" role="button">Unsuspend</a>
+            {{else}}
+            <a href="/suspend?name={{.Name}}" class="btn btn-danger btn-sm" role="button">Suspend</a>
+            / <a href="/unsuspend?name={{.Name}}" class="btn btn-success btn-sm" role="button">Unsuspend</a>
+            {{end}}
```
[the remark this answers is not in the document]
Comment (eze-kiel) [reply webui/assets/home.html:71]: IMO a toolip is not needed and will just add more overhead as the state is indicated on the same row than the button

File: webui/assets/unsuspend.html (hunk 1 above)
Comment (eze-kiel) [reply webui/assets/unsuspend.html:3]: for now they are indeed the same files

File: webui/assets/home.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (eze-kiel) [reply webui/assets/home.html:71]: moreover, I feel that having the same disposition (2 buttons) for all the states provides a better user experience and aesthetics.

It avoids the "Christmas tree" effect (some green, some red, some yellow) by having something consistent and homogeneous, but easily understandable with the transparency layer on the disabled button (and the inability to click on it). Also, it is easier in the case of an "Unknown" state namespace: you can press both the Suspend and Unsuspend buttons to "re-enable" the namespace by setting it a valid annotation, it avoids having a button with a "!" label that forcefully suspends a namespace. In the end, it's the final user who decides, not the developers which is, in term of UX, good practice.

[the remark this answers is not in the document]
Comment (xakraz) [reply webui/assets/home.html:71]: In the end, if it works and solves the initial reported issue, go ahead.
We can keep these feedback for later improvements.

**Review by xakraz — APPROVED** [review]

File: webui/assets/home.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (eze-kiel) [reply webui/assets/home.html:71]: where does this screenshot come from?

**Pull request merged**